### DISCUSSION TITLE
how can i create E2E cypress in single spa tests?

### DISCUSSION THREAD

**Discussion by misaku** [cypress-io/cypress#24383] [Questions and Help]
how can i create E2E cypress in single spa tests?

**Comment by geromegrignon** [comment]
Hi @misaku, please provide more context about the project and your expectations.
What are `single spa tests` in your mind?

**Reply by misaku** [reply]
oh, im sorry.
i would like say project with framowork [single-spa](https://github.com/single-spa/single-spa) to create micro frontends.
im confuse, how can i test my microfrontends with cypress too.

i found this part in [doc](https://single-spa.js.org/docs/testing/e2e)

but i cant understand, and i create  this [question](https://github.com/single-spa/import-map-overrides/issues/71#issuecomment-1292756041) in they git

if you can help with example, i will be glad!
=)

**Reply by geromegrignon** [reply]
Thanks for the details.

The E2E part of Cypress is framework agnostic: it tests an application running in a browser. At this point, it doesn't even know and care how it's built under the hood.
